### PR TITLE
Update devops-command-center.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Added test incident commands for https://github.com/mattermost/mattermost-plugin-incident-management/pull/358 and glossary.

### CODE REVIEW

File: source/administration/devops-command-center.rst [hunk 1]
```diff
@@ -63,13 +63,21 @@ Mattermost Incident Management includes built-in slash commands:
...
+^^^^^^^^^^^^^^^^^^^^
+
+You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insights page.
+
+- ``/incident test create-incident``: This command accepts a playbook ID (that can be chosen from the playbooks the user is a member of, using the autocomplete system), a timestamp, and an incident name. It creates an ongoing incident with the creation date set to the specified timestamp. An example command looks like this: ``/incident test bulk-data 10 3 2020-01-31 2020-11-22 2``.
+- ``/incident test bulk-data``: This command accepts a number of ongoing incidents, a number of ended incidents, a beginning and an end date, and an optional seed. It creates as many ongoing and ended incidents as specified, all of them with their creation date randomly picked between the beginning and end dates. The seed, if available, is used to get reproducible results. The names of the incidents are randomly chosen from a list of incident names and a list of fake company names which are defined in the code. An example command is: ``/incident test create-incident 6utgh6qg7p8ndeef9edc583cpc 2020-11-23 PR-Testing``.
```
Comment (cwarnermm) [inline source/administration/devops-command-center.rst:79]: Consider changing "ended incidents" to "closed incidents" to match Glossary entry available later on the page.

File: source/administration/devops-command-center.rst [hunk 2]
```diff
@@ -176,3 +184,17 @@ Channel Export
 --------------
 
 Please see the `Channel Export plugin documentation <https://mattermost.gitbook.io/channel-export-plugin>`_ for more information.
+
+Glossary
```
Comment (cwarnermm) [inline source/administration/devops-command-center.rst:188]: Consider listing glossary entries in alphabetical order for easy scan and retrieval.

File: source/administration/devops-command-center.rst [hunk 3]
```diff
@@ -176,3 +184,17 @@ Channel Export
...
+--------
+
+- **Incident:** An event requiring the coordinated actions of one or more Mattermost users. An incident is either ongoing or closed.
+- **Playbook:** A task-based process that's followed in order to resolve an incident.
+- **Tasks:** The individual steps required to complete the stages of an incident. Tasks can optionally be assigned to specific incident participants into stages.
+- **Stage:** A set of tasks grouped together to achieve a specific goal of the workflow, which generally need to be completed before proceeding to the next stage of the incident resolution process.
+- **Commander:** The Mattermost user currently responsible for transitioning an incident from ongoing to closed.
+- **Incident channel:** A Mattermost channel dedicated to real-time conversation about the incident.
+- **Incident member:** A Mattermost user with access to the corresponding incident channel.
+- **The RHS:** The incident list and incident details displayed on the right hand side of the webapp. It is not available on mobile.
```
Comment (cwarnermm) [inline source/administration/devops-command-center.rst:198]: Consider updating to clarify the meaning of RHS and remove potential ambiguity: "The incident list and incident details displayed on the right hand side (RHS) of the web app. The RHS is not available on mobile."

File: source/administration/devops-command-center.rst [hunk 4]
```diff
@@ -63,13 +63,21 @@ Mattermost Incident Management includes built-in slash commands:
...
 
 Here are some examples:
 
 - Add a task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
 - One of your tasks may require the header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
 
+Generating test data
+^^^^^^^^^^^^^^^^^^^^
+
+You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insights page.
```
Comment (cwarnermm) [inline source/administration/devops-command-center.rst:76]: Is it "incident insight page" or "incident insights page"? Ensure that all references, including Glossary entries below, reference the page consistently.

File: source/administration/devops-command-center.rst [hunk 5]
```diff
@@ -63,13 +63,21 @@ Mattermost Incident Management includes built-in slash commands:
 Adding slash commands to tasks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Slash commands can be added to tasks to initiate actions as part of your incident response playbook.
+Slash commands can be added to tasks to initiate actions as part of your playbook.
 
 Here are some examples:
 
 - Add a task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
 - One of your tasks may require the header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
```
Comment (cwarnermm) [inline source/administration/devops-command-center.rst:71]: Is it the _channel_ header that would be changed? If so, consider clarifying as: "One of your tasks may require the channel header to be changed to reflect a new status."

File: source/administration/devops-command-center.rst [hunk 6]
```diff
@@ -63,13 +63,21 @@ Mattermost Incident Management includes built-in slash commands:
 Adding slash commands to tasks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Slash commands can be added to tasks to initiate actions as part of your incident response playbook.
+Slash commands can be added to tasks to initiate actions as part of your playbook.
 
 Here are some examples:
 
 - Add a task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``.
```
Comment (cwarnermm) [inline source/administration/devops-command-center.rst:70]: Consider clarifying the type of playbook task: "Add a **communication** task called **Sync up** with the slash command..."

File: source/administration/devops-command-center.rst [hunk 7]
```diff
@@ -63,13 +63,21 @@ Mattermost Incident Management includes built-in slash commands:
...
 
 - Add a task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
 - One of your tasks may require the header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
 
+Generating test data
+^^^^^^^^^^^^^^^^^^^^
+
+You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insights page.
+
+- ``/incident test create-incident``: This command accepts a playbook ID (that can be chosen from the playbooks the user is a member of, using the autocomplete system), a timestamp, and an incident name. It creates an ongoing incident with the creation date set to the specified timestamp. An example command looks like this: ``/incident test bulk-data 10 3 2020-01-31 2020-11-22 2``.
```
Comment (cwarnermm) [inline source/administration/devops-command-center.rst:78]: The example command here likely needs to be switched with the bullet point example below.

File: source/administration/devops-command-center.rst (hunk 1 above)
Comment (cwarnermm) [inline source/administration/devops-command-center.rst:79]: The example command here likely needs to be switched with the bullet point example above for bulk-data

**Review by cwarnermm — CHANGES_REQUESTED** [review]
New slash command examples likely need to be reversed. Additional comments also left in-line for consideration.

File: source/administration/devops-command-center.rst (hunk 6 above)
Comment (justinegeffen) [inline source/administration/devops-command-center.rst:70]:
```suggestion
- Add a communication task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
```

File: source/administration/devops-command-center.rst [hunk 8]
```diff
@@ -63,13 +63,21 @@ Mattermost Incident Management includes built-in slash commands:
...
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Slash commands can be added to tasks to initiate actions as part of your incident response playbook.
+Slash commands can be added to tasks to initiate actions as part of your playbook.
 
 Here are some examples:
 
-- Add a task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
+- Add a communication task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
 - One of your tasks may require the header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
```
Comment (justinegeffen) [inline source/administration/devops-command-center.rst:71]:
```suggestion
- One of your tasks may require the channel header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
```

File: source/administration/devops-command-center.rst [hunk 9]
```diff
@@ -63,12 +63,20 @@ Mattermost Incident Management includes built-in slash commands:
...
 
-- Add a task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
-- One of your tasks may require the header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
+- Add a communication task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
+- One of your tasks may require the channel header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
+
+Generating test data
+^^^^^^^^^^^^^^^^^^^^
+
+You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insights page.
```
Comment (justinegeffen) [inline source/administration/devops-command-center.rst:76]:
```suggestion
You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insight page.
```

File: source/administration/devops-command-center.rst (hunk 1 above)
Comment (justinegeffen) [reply source/administration/devops-command-center.rst:78]: Good catch, thanks. I've checked against the slash command and replaced "closed" with "ended" as the slash command we use is ``/incident end``. I've changed this throughout the doc.

Comment (justinegeffen) [reply source/administration/devops-command-center.rst:78]: Done, thanks!

File: source/administration/devops-command-center.rst [hunk 10]
```diff
@@ -167,12 +175,26 @@ To view information about ongoing incidents, select the **Incidents** icon in th
...
+- **Incident:** An event requiring the coordinated actions of one or more Mattermost users. An incident is either ongoing or ended.
+- **Incident channel:** A Mattermost channel dedicated to real-time conversation about the incident.
+- **Incident insight page:** The incident details and analytics page, which also provides the channel export download link. It is not available on mobile.
+- **Incident member:** A Mattermost user with access to the corresponding incident channel.
+- **Playbook:** A task-based process that's followed in order to resolve an incident.
+- **Playbook configuration page:** The playbook configuration and editing page. It is not available on mobile.
+- **Stage:** A set of tasks grouped together to achieve a specific goal of the workflow, which generally need to be completed before proceeding to the next stage of the incident resolution process.
+- **Tasks:** The individual steps required to complete the stages of an incident. Tasks can optionally be assigned to specific incident participants into stages.
+- **The RHS:** The incident list and incident details displayed on the right hand side of the webapp. It is not available on mobile.
```
Comment (justinegeffen) [inline source/administration/devops-command-center.rst:200]:
```suggestion
- **The RHS:** The incident list and incident details displayed on the right hand side (RHS) of the web app. It is not available on mobile.
```

Comment (justinegeffen) [inline source/administration/devops-command-center.rst:200]:
```suggestion
- **The RHS:** The incident list and incident details displayed on the right hand side (RHS) of the web app. It is not available on mobile.
```

**Review by agarciamontoro — CHANGES_REQUESTED** [review]
Yay! :tada: Just a minor change below.

I assume this is in the page for administrators, right? The end user will not be very interested in this.

**Comment by justinegeffen** [comment]
Thanks, @agarciamontoro! I don't see a suggestion/change - could you let me know what it was? :)

File: source/administration/devops-command-center.rst [hunk 11]
```diff
@@ -63,12 +63,20 @@ Mattermost Incident Management includes built-in slash commands:
...
+^^^^^^^^^^^^^^^^^^^^
+
+You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insight page.
+
+- ``/incident test bulk-data``: This command accepts a number of ongoing incidents, a number of ended incidents, a beginning and an end date, and an optional seed. It creates as many ongoing and ended incidents as specified, all of them with their creation date randomly picked between the beginning and end dates. The seed, if available, is used to get reproducible results. The names of the incidents are randomly chosen from a list of incident names and a list of fake company names which are defined in the code. An example command is: ``/incident test create-incident 6utgh6qg7p8ndeef9edc583cpc 2020-11-23 PR-Testing``.
+- ``/incident test create-incident``: This command accepts a playbook ID (that can be chosen from the playbooks the user is a member of, using the autocomplete system), a timestamp, and an incident name. It creates an ongoing incident with the creation date set to the specified timestamp. An example command looks like this: ``/incident test bulk-data 10 3 2020-01-31 2020-11-22 2``.
```
Comment (agarciamontoro) [inline source/administration/devops-command-center.rst:79]: The examples here are swapped!

**Review by agarciamontoro — CHANGES_REQUESTED** [review]
Oops, it seems I forgot to add the comment. There it is!

File: source/administration/devops-command-center.rst (hunk 11 above)
Comment (justinegeffen) [reply source/administration/devops-command-center.rst:79]: Yep, @cwarnermm suggested swapping them which I did but I should have confirmed whether they're in a specific order? If they are, I will swap them around and add a note indicating that one needs to be used prior to the other.

Comment (cwarnermm) [reply source/administration/devops-command-center.rst:78]: Good catch on order of operations for running the commands. In addition, the command examples appear to be for the opposite command where ``/incident test create-incident has an example for bulk-data and vice-versa.

**Review by cwarnermm — COMMENTED** [review]
Please review command examples.

**Review by cwarnermm — APPROVED** [review]
Looks good!

File: source/administration/devops-command-center.rst (hunk 11 above)
Comment (agarciamontoro) [reply source/administration/devops-command-center.rst]: > the command examples appear to be for the opposite command where ``/incident test create-incident has an example for bulk-data and vice-versa

Oh yeah, that's what I meant, sorry for the confusion!

**Review by agarciamontoro — APPROVED** [review]
:tada:

**Pull request merged**